examples/README.md

```diff
@@ -19,21 +19,23 @@
 2. Install `kustomize` v3.1.0+ (see [here](https://github.com/kubernetes-sigs/kustomize/releases)
 
 ## Generation
-For convenience, a generation script which populates templates based on openstack cloud provider
-configuration is provided.
+
+For convenience, a script is provided which generates example YAMLs. The generated YAMLs then have to be customized manually.
 
 1. Run the generation script.
+
 ```
 cd examples
 export CLUSTER_NAME=<cluster-name>
-./generate.sh [options] <path/to/clouds.yaml> <openstack cloud> [output folder]
+./generate.sh [options] <path/to/clouds.yaml> <openstack-cloud> [output folder]
 ```
 
    `<clouds.yaml>` is a YAML configuration file for Openstack, refer to [clouds.yaml](https://github.com/kubernetes-sigs/cluster-api-provider-openstack/blob/master/pkg/cloud/openstack/clients/clouds.yaml), and [OpenStack client configuration files](https://docs.openstack.org/python-openstackclient/latest/configuration/index.html#configuration-files).
```

Comment:
I am not sure if this sample cloud.yaml exist in master branch. This should be removed or maybe add sample clouds.yaml to /pkg/cloud/services

Reply:
yes, seems it doesn't exit here

Reply:
Oh yeah, you're right. The file did exist a while ago. I'd like to remove the referral here. I think the link to the OpenStack doc (which also contains a example cloud.yaml) is good enough